.github/workflows/image_compression.yml

```diff
@@ -0,0 +1,49 @@
+name: Image Compression Check
```

Comment:
This workflow isn't running on the current PR. Can you make it run (possibly temporarily) so that we can see the result? It should pass.